src/Ses/Input/SendEmailRequest.php

```diff
@@ -127,26 +127,104 @@ public function getReplyToAddresses(): array
+
+        (static function ($input) use (&$payload) {
+            if (null !== $v = $input->getSimple()) {
+                (static function ($input) use (&$payload) {
+                    (static function ($input) use (&$payload) {
+                        $payload['Content.Simple.Subject.Data'] = $input->getData();
+                        if (null !== $v = $input->getCharset()) {
+                            $payload['Content.Simple.Subject.Charset'] = $v;
+                        }
+                    })($input->getSubject());
+                    (static function ($input) use (&$payload) {
+                        if (null !== $v = $input->getText()) {
+                            (static function ($input) use (&$payload) {
+                                $payload['Content.Simple.Body.Text.Data'] = $input->getData();
+                                if (null !== $v = $input->getCharset()) {
+                                    $payload['Content.Simple.Body.Text.Charset'] = $v;
+                                }
+                            })($v);
+                        }
+                        if (null !== $v = $input->getHtml()) {
+                            (static function ($input) use (&$payload) {
+                                $payload['Content.Simple.Body.Html.Data'] = $input->getData();
+                                if (null !== $v = $input->getCharset()) {
+                                    $payload['Content.Simple.Body.Html.Charset'] = $v;
+                                }
+                            })($v);
+                        }
+                    })($input->getBody());
+                })($v);
+            }
+            if (null !== $v = $input->getRaw()) {
+                (static function ($input) use (&$payload) {
+                    $payload['Content.Raw.Data'] = base64_encode($input->getData());
+                })($v);
+            }
+            if (null !== $v = $input->getTemplate()) {
+                (static function ($input) use (&$payload) {
+                    if (null !== $v = $input->getTemplateArn()) {
+                        $payload['Content.Template.TemplateArn'] = $v;
+                    }
+                    if (null !== $v = $input->getTemplateData()) {
+                        $payload['Content.Template.TemplateData'] = $v;
+                    }
+                })($v);
+            }
+        })($this->Content);
+        (static function ($input) use (&$payload, $indices) {
+            $indices->k7c3bed1 = 0;
+            foreach ($input as $value) {
+                ++$indices->k7c3bed1;
+                (static function ($input) use (&$payload, $indices) {
+                    $payload["EmailTags.{$indices->k7c3bed1}.Name"] = $input->getName();
+                    $payload["EmailTags.{$indices->k7c3bed1}.Value"] = $input->getValue();
+                })($value);
+            }
+        })($this->EmailTags);
+
+        if (null !== $v = $this->ConfigurationSetName) {
```

Comment:
Oh, This code is not super pretty.

Reply:
Agreed :/

I'm open to any suggestion

Reply:
I don’t have any...
Let’s merge and improve this later.